src/modules/layout/components/Header.js

```diff
@@ -144,7 +152,8 @@ class Header extends React.PureComponent<PropsType, StateType> {
           </HorizontalLeft>
           <MaterialHeaderButtons>
             <Item title='Search' iconName='search' onPress={this.showSearchBar} />
-            <Item title='Change Language' iconName='language' onPress={this.goToLanguageChange} />
+            <Item title='Change Language' iconName='language'
+                  onPress={isOtherLanguageAvailable ? this.goToLanguageChange : null} />
```

Comment:
Should we display any kind of information that there is no translation for the current page?

Reply:
I think it is fine if the button is greyed-out and not clickable. At least I dont know how to give proper feedback.
Maybe a shaking-animation: http://www.mircozeiss.com/shake-that-login-form-with-angularjs

but this it not important and can be done in a separate task

Reply:
I think we could also just not care if no other language is available and still open the modal -- I guess this is a rare case...

Reply:
I don't care. It could also be greyed out and show a toast on click.

Reply:
Alright I just removed it and we won't care.